docs/dev/zksync-v2/bridging-funds.md

```diff
@@ -1,3 +1,52 @@
 # Bridging funds
 
-This section is under development. It will be updated soon.
+## Introduction
+
+Bridging is implemented by having two contracts 
+(one deployed to L1, and the second deployed to L2)
+communicating with each other using [L1 <-> L2 interoperability](./l1-l2-interop.md).
+
+There are two dedicated transaction types for native token bridging:
+
+- `Deposit` for bridging tokens to zkSync.
+- `Withdraw` for bridging tokens from zkSync.
+
+Developers are free to build their own bridge for any token.
+However, we are providing our default bridges (one for ETH and one for ERC20 tokens), which can be used for developers to use basic bridging.
+
+::: warning
+
+Addresses of tokens on L2 will always differ from L1 addresses.
+
+:::
+
+
+## Default bridges
+
+You can get default bridges' addresses using `zks_getBridgeContracts` endpoint
+or `getDefaultBridgeAddresses` method of `Provider` in our JS SDK.
+
+### Deposits
+
+User makes a transaction to the L1 bridge by calling the `deposit` method. The following actions are then being executed:
+
+- L1 tokens are being sent to the L1 bridge and become locked there.
+- L1 bridge initiates a transaction to the L2 bridge using L1 -> L2 communication.
+
+Within the L2 transaction tokens are being minted and sent to the specified address on L2.
+
+If this transaction fails for any reason (for example, the provided fee is too low)
+then the L2 -> L1 message is being sent.
+In this case the inclusion of the message can be proven on the L1 bridge to return the deposited funds back.
+
```

Comment:
I suggest to add here (3/3):
The log message described above is not supported yet on our current testnet.

Reply:
The log message itself is supported but there is no tooling for users. So in theory, users can use this feature by manually getting proof from ZkSync contract and then verifying it on the bridge. 

But I guess saying "The log message described above is not supported yet on our current testnet." until SDK has this feature is better than explaining all that.

Reply:
oh, in that case you can just say that our SDK doesn't support it, but it's not really critical